### DISCUSSION TITLE
Cannot extract files

### DISCUSSION THREAD

**Discussion by cakueayam** [lllyasviel/Fooocus#3494] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/e721d9c2-e272-462c-b645-46bb1d6b371f)
it's been like this for 3 hours, what happen?

**Comment by Rogala** [comment, marked as answer]
The options are many:
1 The file is not fully loaded;
2 The archiver did not have enough system resources;
3 There is not enough space on the disk where the unzipping is going.

Use [7-ZIP](https://www.7-zip.org/)

**Reply by cakueayam** [reply]
thanks for the reply, i already download 7 zip and there's no otion to extract using 7 zip, i click on run realistic bat. is that correct?

**Comment by Ak075-Artist** [comment]
I got the same error. After a few tries, I used WinRar extract, and its done.
